packages/status-page/src/pages/home/Home.svelte

```diff
@@ -99,6 +107,7 @@
         if (value.toString() === "true") return "red";
         return "green";
       },
+      tooltip: "Whether the Taiko smart contract on Layer 1 has been halted",
```

Comment:
```suggestion
      tooltip: "Boolean describing whether the Taiko smart contract on Layer 1 has been halted.",
```

Reply:
I don't think using 'boolean' is a good idea here

Reply:
I do missing sometimes tho